python/chronos/src/bigdl/chronos/autots/autotsestimator.py

```diff
@@ -338,3 +339,20 @@ def get_best_config(self):
         :return: A dictionary of best hyper parameters
         """
         return self.model.get_best_config()
+
+    def _n_sampling(self, n_sampling):
```

Comment:
the method should be clearer in name, may be we can call it `_recalculate_n_sampling`

Reply:
btw, we may put this function to a util file (i.e. chronos/autots/utils.py) since automodel and autotsest all use it.